pkg/authorization/api/register.go

```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-const GroupName = ""
+const GroupName = "authorization.openshift.io"
```

Comment:
> @deads2k so we need the kube PR to land first on order to be able to merge this?

Can we generate the clients using the `""` group for now, then generate another set (we'll need both anway) after we register with both names.  

We divvied up groups so we'd know whether any shouldn't be generated single folder.